OSDOCS-11759: Adds information about OpenShift Lightspeed to ocp web console docs (#81581)
openshift/openshift-docs: modules/openshift-lightspeed-web-console.adoc

// Module included in the following assemblies:
//
// * capabilities-web-console.adc

:_mod-docs-content-type: CONCEPT
[id="openshift-lightspeed-web-console_{context}"]

> =  Enhancing your experiance in the {product-title} web console by using {ols-official}

Reviewer: ```suggestion
=  Enhancing your experience in the {product-title} web console by using {ols-official}
```

Reviewer 2: Maybe the title could be simplified as "Red Hat OpenShift Lightspeed in the web console", to keep consistent with other following capabilities, such as:
"Red Hat OpenShift Pipelines in the web console"
@opayne1 WDYT?

> Red Hat OpenShift Lightspeed is a generative artificial intelligence powered virtual assistant for {product-title}. Lightspeed functionality uses a natural-language interface in the OpenShift web console.

Reviewer: suggestion to use variables here:
```suggestion
{ols-official} is a generative artificial intelligence powered virtual assistant for {product-title}. Lightspeed functionality uses a natural-language interface in the {product-title} web console.
```

Reviewer: i also think maybe "ai powered" might need a hyphen when it's an adjective:
```suggestion
Red Hat OpenShift Lightspeed is a generative artificial intelligence-powered virtual assistant for {product-title}. Lightspeed functionality uses a natural-language interface in the OpenShift web console.
```
